cli/src/commands/services.js

```diff
@@ -95,87 +113,155 @@ export async function dockerComposeUp({ verbose = false, exclude = [], ci = fals
+            failSpinner(
+                spinner,
+                `${chalk.red.bold(
+                    'ERROR:',
+                )} Something went wrong. Check the logs above for more information ☝️, or try inspecting the logs with ${chalk.red.cyan(
+                    'botfront logs',
+                )}.`,
+            );
             console.log(e);
         } else {
             stopSpinner(spinner);
-            failSpinner(spinner, 'Couldn\'t start Botfront. Retrying in verbose mode...', { exit: false });
-            return dockerComposeUp({ verbose: true, exclude: exclude }, workingDir, null, spinner);
+            failSpinner(spinner, 'Couldn\'t start Botfront. Retrying in verbose mode...', {
+                exit: false,
+            });
+            return dockerComposeUp(
+                { verbose: true, exclude: exclude },
+                workingDir,
+                null,
+                spinner,
+            );
         }
     }
 }
 
 export async function dockerComposeDown({ verbose }, workingDir) {
-    if (workingDir) shell.cd(workingDir)
+    if (workingDir) shell.cd(workingDir);
     if (!isProjectDir()) {
         const noProject = chalk.yellow.bold('No project found in this directory.');
         const killall = chalk.cyan.bold('botfront killall');
-        const noProjectMessage = (
+        const noProjectMessage =
             `${noProject}\n\nIf you don't know where your project is running from,\n` +
-            `${killall} will find and shut down any Botfront\nproject on your machine.`
-        );
+            `${killall} will find and shut down any Botfront\nproject on your machine.`;
         return console.log(boxen(noProjectMessage));
     }
-    const spinner = ora('Stopping Botfront...')
+    const spinner = ora('Stopping Botfront...');
     spinner.start();
     let command = 'docker-compose down';
-    await shellAsync(command, { silent: !verbose })
+    await shellAsync(command, { silent: !verbose });
     spinner.succeed('All services are stopped. Come back soon... 🤗');
 }
 
 export async function dockerComposeStop(service, { verbose }, workingDir) {
-    await dockerComposeCommand(service, {name: 'stop', action: 'stopping'}, verbose, workingDir )
-        .catch(consoleError)
+    await dockerComposeCommand(
+        service,
+        { name: 'stop', action: 'stopping' },
+        verbose,
+        workingDir,
+    ).catch(consoleError);
+}
+
+export async function setProject(bf_project_id) {
+    if (!bf_project_id || typeof bf_project_id !== 'string') {
+        throw new Error(`Project ID '${bf_project_id}' is not valid.`);
+    }
+    const { services } = getComposeTemplateFile();
+    const { services: actualServices } = getGeneratedComposeFile();
+    const exclude = Object.keys(services).filter((k) => !(k in actualServices));
+    updateProjectFile({ env: { bf_project_id } });
```

Comment:
`updateProjectFile` has` enableMongoAuth = true` as a default argument, so here if you passed no auth at the init this call will add auth. Moreover it will also change the auth every time you change project

https://user-images.githubusercontent.com/27060431/106520869-6749b100-64ab-11eb-9276-a21e63b3707f.mov

I'm not sure but it might also change

Reply:
Good catch!